docs/setup_gdb.md

```diff
@@ -92,21 +92,22 @@ We recommend enabling the address sanitizer and undefined behavior sanitizer. Th
 
 First, edit your `Makefile` and add the `CXXFLAGS` recommended by the [ASAN Quick Start](setup_asan.html#quick-start).
 
-### Arguments and options
-Arguments and options are inputs to a program typed at the command line.  Here's an example from EECS 280 Project 4:
+### Command-Line Arguments
```

Comment:
Do you think that adding "Options" will help avoid confusion when they are used in the future?

```suggestion
### Command-Line Arguments and Options
```